### PR TITLE
docs(react-accordion): cleanup examples & docs

### PR DESCRIPTION
**Pull request by layershifter**
See comments on the files 👇

### CODE REVIEW

File: packages/react-components/react-accordion/src/components/Accordion/Accordion.types.ts [hunk 1]
```diff
@@ -35,30 +35,32 @@ export type AccordionToggleData = {
 
 export type AccordionProps = ComponentProps<AccordionSlots> & {
   /**
-   * Default value for the uncontrolled state of the panel
+   * Default value for the uncontrolled state of the panel.
```
Comment (layershifter) [inline packages/react-components/react-accordion/src/components/Accordion/Accordion.types.ts:38]: Added dots everywhere to align with other comments.

File: packages/react-components/react-accordion/src/stories/Accordion.stories.tsx [hunk 2]
```diff
@@ -25,10 +25,6 @@ export default {
     },
   },
   argTypes: {
-    inline: {
```
Comment (layershifter) [inline packages/react-components/react-accordion/src/stories/Accordion.stories.tsx:28]: - These props are not for `Accordion` 
- They are displayed without JSDoc
   image

File: packages/react-components/react-accordion/src/components/Accordion/Accordion.types.ts [hunk 3]
```diff
@@ -35,30 +35,32 @@ export type AccordionToggleData = {
...
   /**
-   * Indicates if keyboard navigation is available and gives two options,
-   * linear or circular navigation
+   * Indicates if keyboard navigation is available and gives two options, linear or circular navigation.
    */
   navigation?: 'linear' | 'circular';
 
+  /**
+   * Callback to be called when the opened items change.
+   */
```
Comment (layershifter) [inline packages/react-components/react-accordion/src/components/Accordion/Accordion.types.ts:59]: New comment.

File: packages/react-components/react-accordion/src/stories/AccordionCollapsible.stories.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import { AccordionItem, AccordionHeader, AccordionPanel, Accordion, AccordionProps } from '../index';
+import { AccordionItem, AccordionHeader, AccordionPanel, Accordion } from '../index';
 
-export const Collapsible = (args: AccordionProps) => (
-  <Accordion {...args}>
+export const Collapsible = () => (
+  <Accordion collapsible>
```
Comment (layershifter) [inline packages/react-components/react-accordion/src/stories/AccordionCollapsible.stories.tsx:5]: The same as #23231. Our export to CodeSandbox does not support `.args` => all examples look the same as `args` is `undefined`.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4922a8a8b9e52ef8cc7f299771d983a6a09fa861'>4922a8a8b9e52ef8cc7f299771d983a6a09fa861</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/Avatar-mount_base.html">900</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/Avatar-mount.html">944</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/Button-mount_base.html">574</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/Button-mount.html">566</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/FluentProvider-mount_base.html">1824</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/FluentProvider-mount.html">1856</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/FluentProviderWithTheme-mount_base.html">274</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/FluentProviderWithTheme-mount.html">298</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">240</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">236</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">315</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">315</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/MakeStyles-mount_base.html">1601</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23235/perf-test-react-components/MakeStyles-mount.html">1594</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Pull request merged**